sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/files/arm64_defconfig-6.1

```diff
@@ -32,6 +32,8 @@ CONFIG_CRYPTO_SHA1_ARM64_CE=y
 CONFIG_CRYPTO_SHA2_ARM64_CE=y
 # CONFIG_DEBUG_PREEMPT is not set
 CONFIG_DM_DEBUG=y
+CONFIG_DRM=m
```

Comment:
Thank you, I would even suggest to add this to the common config so that this features is consistently available on both architectures.

Reply:
sure, will add it on the amd64 if it works on the arm64.

Reply:
Please move all lines to `sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/files/commonconfig-6.1`